scripts/format_arpa_lm.py

```diff
@@ -44,6 +46,9 @@
 ngram_order = int(f.readline())
 f.close()
 
+# set the memory restriction for "sort"
+sort_mem_opt = ("--buffer-size={0} ".format(args.max_memory))
```

Comment:
when args.max_memory == '', the sort_mem_opt should be '--buffer-size= '. I'm not sure whether sort would complain If we pass this sort_mem_opt to it.
Is the magic of outter parentheses on the right side makes the sort_mem_opt to empty?